### ISSUE TITLE
Top Status Bar does not stretch to screen width

### ISSUE THREAD

**Issue by Slicerin** [FireSiku/LUI-V4#7]
<img width="1919" alt="top bar not stretching" src="https://user-images.githubusercontent.com/41658720/43217049-23326da2-901b-11e8-9567-786f9accbe0a.PNG">

**Steps to Reproduce**

Login to a character
Take note of the information on the top bar

**Observed Results**

The status bar does not stretch to screen size (This was done on 4K resolution, not tested on 1080p)

**Comment by FireSiku** [comment]
Yeah, definitely a thing I needed to come back and fix after I noticed it when I went testing something unrelated.

Changing title to convey the issue at hand.
